packages/ui/src/views/repo/pull-request/details/components/changes/pull-request-changes.tsx

```diff
@@ -124,16 +124,8 @@ const LineTitle: React.FC<LineTitleProps> = ({
         </div>
 
         <div className="flex items-center gap-x-1">
-          {addedLines != null && addedLines > 0 && (
-            <StatusBadge variant="outline" size="sm" theme="success">
-              +{addedLines}
-            </StatusBadge>
-          )}
-          {deletedLines != null && deletedLines > 0 && (
-            <StatusBadge variant="outline" size="sm" theme="danger">
-              -{deletedLines}
-            </StatusBadge>
-          )}
+          {addedLines != null && addedLines > 0 && <CounterBadge theme="success">+{addedLines}</CounterBadge>}
```

Comment:
Not an issue, but you can simplify a wee bit.
```suggestion
          {!!addedLines && <CounterBadge theme="success">+{addedLines}</CounterBadge>}
```